client/src/Authorization/Login/Login.js

```diff
@@ -0,0 +1,127 @@
+import React, { Component } from "react";
+import { Redirect } from "react-router-dom";
+import { authService } from "../_auth-services/authentication";
+import "./Login.css";
+import bookImage from "../../pictures/books.png";
+
+class Login extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            username: "",
+            password: "",
+            submitted: false, 
+            redirect: false
+        };
+    }
+    
+    redirectToMainPage = () => {
+        this.setState({redirect: true});
+    }
+
+    componentDidMount() {
+        if(authService.isAuthenticated()) {
+            this.redirectToMainPage();
+        }
+    }
+
+    loginHandler = async event => {
+        const { username, password } = this.state;
+
+        event.preventDefault();
+        this.setState({ submitted: true });
+
+        if (!(username && password)) return;
+
+        const response = await authService.login(username, password);
+        if(response.success) {
+            const user = {
+                name: username,
+                token: response.token
+            }
+            authService.authenticateUser(user);
+            this.redirectToMainPage();
```

Comment:
After successful sign in you can redirect to url /signedin and add empty route in routes.js ( e.g. <Route path='/signedin' />)